packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_create_loan_scheme.test.ts

```diff
@@ -0,0 +1,132 @@
+import { DeFiDRpcError, GenesisKeys } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import { fundEllipticPair, sendTransaction } from '../test.utils'
+import { WIF } from '@defichain/jellyfish-crypto'
+import { BigNumber } from '@defichain/jellyfish-json'
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+
```

Comment:
Missing describe block

Reply:
Copied the format from AppointOracle but it is okay. I will add the describe block. The code review for this PR will set the foundation for the rest of my DFTXs.

Reply:
Great, you should use the jellyfish-testing framework as well then

Reply:
Read other's code, `testing.container.generate(1) `should be `testing.generate(1)`